src/components/inline-loading/_inline-loading.scss

```diff
@@ -0,0 +1,61 @@
+@import '../../globals/scss/vars';
+@import '../../globals/scss/colors';
+@import '../../globals/scss/css--reset';
+@import '../../globals/scss/import-once';
+@import 'keyframes';
+
+@include exports('inline-loading') {
+  .#{$prefix}--inline-loading {
+    display: flex;
+    width: 100%;
+    align-items: center;
+  }
+
+  .#{$prefix}--inline-loading__text {
+    font-size: 0.875rem;
```

Comment:
Can you just switch this to `@include typescale('zeta');` in case there are future updates to the type scale?